packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/semanticdata/repositories/ISemanticDataRepository.java

```diff
@@ -60,4 +61,13 @@ WITH RECURSIVE dependencies AS (
         SELECT dependency_semantic_data_id FROM dependencies
         """)
     List<UUID> findAllDependenciesRecursivelyById(UUID id);
+
+    @Query("""
+            SELECT semanticDataDomain.*
+            FROM semantic_data semanticData
+            JOIN semantic_data_domain semanticDataDomain
+            ON semanticData.id = semanticDataDomain.semantic_data_id
+            WHERE semanticData.id = :id
+            """)
+    List<SemanticDataDomain> findDomainsById(UUID id);
```

Comment:
The goal of the AggregateRoot and the repository is to explicitly be the entry point loaded from the repository. We could on the other hand provide an operation like `boolean isUsingDomains(List<String> domainUris)` which would accomplish the same goal.

Reply:
Done.